Ellielee308/wits-travel
# Feature/edit form upload images

[Website URL](https://witz-ec201--ellie-suf90tsc.web.app/backstage/spot-manage)

## Description
1. Image Upload: Users can now upload new main images and additional images while editing spots.
2. Heart Icon Interaction: The heart icon will now be filled when clicked, with a jumping animation to provide visual feedback.
3. Input Field Changes: The transportation input field has been changed from an input box to a textarea for better user interaction.
Additional minor fixes include removing unnecessary comments and refining the onSubmit function.

## Test Plan
1. Image Upload:
- Click the "edit" button of any spot.
- Upload a new main image or additional images.
- Submit the form and verify that the page updates accordingly.
- Check Firestore to ensure that the stored data is correct.
- Check Firebase Storage to confirm that images in the spotImages directory have been updated as expected.

2. Heart Icon Interaction:
- Click on the heart icon and verify that it is filled and displays a jumping animation.

3. Input Field Change:
- Confirm that the transportation field is now a textarea and accepts multiline input.
